tree/ntupleutil/v7/inc/ROOT/RNTupleImporter.hxx

```diff
@@ -237,9 +242,13 @@ private:
    void ResetSchema();
    /// Sets up the connection from TTree branches to RNTuple fields, including initialization of the memory
    /// buffers used for reading and writing.
-   RResult<void> PrepareSchema();
+   RResult<void> PrepareSchema(TTree *sourceTree);
```

Comment:
Do we mean 'Schema' or 'Model' here?

Reply:
I think 'Model', is that correct @jblomer?